Improve MySQL Data Connector quick start (#160)
spiceai/quickstarts: mysql/README.md

Invoke MySQL in the prompt.

```bash
mysql -u USERNAME -pPASSWORD
```

> Inside MySQL, create and use a sample MySQL database named `spice_demo`.

Reviewer: ```suggestion
Using the MySQL console, create a sample database named `spice_demo`.
```
